### DISCUSSION TITLE
Radius for panels, not a token?

### DISCUSSION THREAD

**Discussion by koesper** [primefaces/community#831] [Figma UI Kit] [answered]
I'm just getting started with the UI Kit (v2.2.0), and am trying to change some tokens to match the design changes i've already done in my PrimeNG implementation.

One of the changes is the border radius, as used by card elements.
It appears that the various `panel` type elements in the UI-kit have hardcoded values for the frame radius?
eg:
panel > card > demo > simple card > card, has radius `6` on the frame
panel > accordion > accordion > _accordion-tab, has radius `6` on the frame

Shouldnt these be filled with `{global.borderRadius}`?

Is this just not filled in UI Kit, or just not possible within Figma?

**Comment by onursenture** [comment, marked as answer]
Correct. It would be ideal for the border-radius to be defined as a token so it can be changed and controlled from a common place and linked to `{global.borderRadius}` if it is not overridden in exceptional cases with values like `50%`, `0`.

We have noted this for our upcoming releases, and thank you for catching this and informing us.

Note that the organizational structure of tokens may change in the next release to align with the new theme engine we're developing for UI libraries.
